cyclonedx/model/bom.py

```diff
@@ -133,7 +149,8 @@ def from_parser(parser: BaseParser) -> 'Bom':
         bom.add_components(parser.get_components())
         return bom
 
-    def __init__(self) -> None:
+    def __init__(self, components: Optional[List[Component]] = None, services: Optional[List[Service]] = None,
+                 external_references: Optional[List[ExternalReference]] = None) -> None:
```

Comment:
please use enforce kwargs so the class is kind of future-prove
```path
- def __init__(self, components: Optional[List[Component]] = None, services: Optional[List[Service]] = None,
-                   external_references: Optional[List[ExternalReference]] = None) -> None:
+ def __init__(self, *, 
+                    components: Optional[List[Component]] = None, services: Optional[List[Service]] = None,
+                    external_references: Optional[List[ExternalReference]] = None) -> None:
```

Reply:
Have adopted PEP-3102 now as we previously discussed in all model classes.